scico/linop/radon_svmbir.py

```diff
@@ -190,20 +192,28 @@ class SVMBIRWeightedSquaredL2Loss(WeightedSquaredL2Loss):
     of :class:`scico.linop.Diagonal`. If :math:`W` is None, it is set to
     :class:`scico.linop.Identity`.
 
+    The extended loss differs from a typical weighted squared :math:`\ell_2` loss
+    is the following ways.
+
     When ``positivity=True``, the prox projects onto the non-negative
     quadrant, but the the loss, :math:`\alpha l(\mb{y}, A(\mb{x}))`,
     is unaffected by this setting and still evaluates to finite values
     when :math:`\mb{x}` is not in the non-negative quadrant.
+
+    When the ``is_masked`` option of the associated ParallelBeamProjector is True,
+    the reconstruction is computed over a masked region of the image as described
```

Comment:
Consider adding sphinx `:class:` cross-referencing so that `ParallelBeamProjector` links to the corresponding docs page for that class.

Reply:
Fixed.